.misc/build_and_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Because this file is bound to travis, the filename should maybe contain "travis" somehow

Reply:
Accepted